index.js

```diff
@@ -11,10 +13,11 @@ class DkgClient {
         const baseServiceManager = new BaseServiceManager(config);
         const services = baseServiceManager.getServices();
 
-        this.asset = new AssetOperationsManager(config, services);
-        this.blockchain = new BlockchainOperationsManager(config, services);
```

Comment:
Why are we removing blockchain operations manager from public API?

Reply:
We don't have blockchain operations manager implementation

Reply:
Ok, my bad, didn't pull the changes from the master